runners/google-cloud-dataflow-java/worker/legacy-worker/build.gradle

```diff
@@ -213,7 +206,7 @@ dependencies {
     compile project(":sdks:java:fn-execution")
     compile project(path: ":runners:google-cloud-dataflow-java:worker:windmill", configuration: "shadow")
     shadow library.java.vendored_guava_26_0_jre
-    compile "org.conscrypt:conscrypt-openjdk:2.5.1:linux-x86_64"
+    compileOnly "org.conscrypt:conscrypt-openjdk:2.5.1:linux-x86_64"
```

Comment:
Can you comment here:

 - why this unusual configuration? (because jar signing)
 - why is it OK? (because gRPC brings it in)

Later, we can include it in the container instead of getting lucky with transitive deps.

Reply:
Yep that's a good idea. Done!